inc/saiswitch.h

```diff
@@ -60,6 +60,12 @@ typedef enum _sai_switch_oper_status_t
     /** Switch encountered a fatal error */
     SAI_SWITCH_OPER_STATUS_FAILED,
 
+    /** VOQ Switch in critical state with available fabric ports below configured configured threshold */
+    SAI_SWITCH_OPER_STATUS_VOQ_FABRIC_PORT_THRESHOLD_LOW,
```

Comment:
If we choose to rename SAI_SWITCH_ATTR_FABRIC_PORT_THRESHOLD_LOW, _HIGH (see comments below), then we could similarly rename this as well.

Reply:
https://github.com/opencomputeproject/SAI/pull/1793/commits/64d77e084762eeb7ee35ac1ab1641b8584104c6d